packages/webpack/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@lavamoat/webpack",
-  "version": "0.1.0-beta.0",
+  "version": "0.1.1-beta.0",
```

Comment:
@naugtur I don't think this is right.

Reply:
It went from 0.0.1 to 0.1.0 before. It's not what I intended but as long as it's `0.x` I don't mind skipping the effort to fix it.